docs/doc/01-cloud/20-bi-tool/00-metabase.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Connect Databend Cloud To Metabase
+sidebar_label: Metabase
+description: 
+  Connect Databend Cloud To Metabase
+---
+
+[Metabase](https://www.metabase.com/) is a simple but powerful analytics tool that enables everyone to make business decisions using their data without requiring any technical skills.
+
+Before connecting to Metabase, sign in to your account on [Databend Cloud](../../10-deploy/20-databend-cloud.md).
+
+## Install Metabase
+
+Metabase is a Java application that can be executed by simply downloading the [JAR file](https://www.metabase.com/start/oss/jar). To install Metabase, simply run this command:
+
+> CAUTION: Metabase requires that you have Java 8 or higher available on your system.
```

Comment:
Note: Metabase requires that you have Java 8 or higher versions installed on your system.